phpcs.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WordPress Coding Standards" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+
+  <!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+  <!-- See https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/.phpcs.xml.dist -->
+
+  <!-- Set a description for this ruleset. -->
+  <description>Coding standards used for checking the Twenty Twenty theme.</description>
+
+  <!-- Exclude patterns -->
+  <exclude-pattern>/vendor/*</exclude-pattern>
+  <exclude-pattern>/node_modules/*</exclude-pattern>
+
+  <!-- Include the WordPress ruleset, with space for exclusions if necessary. -->
+  <rule ref="WordPress-Extra">
```

Comment:
Rather than adding an ad-hoc set of exclusions, these should be added in followup PRs, with appropriate rationale.

Reply:
Ok, will add the `Extra` and then excluded when needed and the situation demands it. I used the `twentynineteen` as a 'template' for what to use, and this was added there, so I thought it couldn't hurt 🙂

Reply:
I concur with @pento. I'd start with excluding nothing. Twenty-Twenty should - IMO - encompass all best practices.

If, while fixing the issues, it turns out a particular rule is not realistic to include, it can be excluded later or selectively excluded, i.e. excluded just for a particular group of files, not for the whole theme.

It would also be good to include inline comments in the ruleset documenting why certain rules are excluded.

Also, for documentation purposes, it would be better to have exclude rules within the ruleset which added them, so if the last three (`Squiz.Commenting....`) would be kept, they should be moved to the `WordPress-Docs` ruleset.